src/app/core.py

```diff
@@ -221,7 +221,8 @@ def ntia_check_helper(request):
                 """ If any warnings are returned """
                 if (request.is_ajax()):
                     ajaxdict["type"] = "warning"
-                    ajaxdict["data"] = "The following warning(s) were raised:\n" + str(retval)
+                    warnings = str(retval)
+                    ajaxdict["data"] = "The following warning(s) were raised:\n" + warnings.replace('\n', '<br />')
```

Comment:
Just a nit - it would be nice to keep a linefeed in case anyone looks at the raw HTML.

Reply:
We could also have
```
"The following warning(s) were raised:<br />\n"
```
instead of:
```
"The following warning(s) were raised:\n"
```
so that the first warning is not on the same line.